Runtime/Scripts/MobiledgeXIntegration.cs

```diff
@@ -225,6 +233,46 @@ public AppPort GetAppPort(LProto proto, int port = 0)
             }
         }
 
+        /// <summary>
+        /// Updates the public port, if necessary, if this AppPort is in Mel Mode.
+        /// </summary>
+        private bool AppPortForMel(FindCloudletReply reply, LProto proto, int defaultPort)
+        {
+            if (IsNetworkDataPathEdgeEnabled() && melMessaging.IsMelEnabled())
+            {
+                if (reply.ports.Length > 1)
+                {
+                    throw new Exception("MobiledgeX: Unexpected Port length for MEL mode.");
+                }
+
+                AppPort appPort = reply.ports[0];
```

Comment:
Does this appPort need to go back into reply.ports after it is modified?

Reply:
In place modify. The original object is empty to begin with.